store/types/store.go

```diff
@@ -38,6 +38,48 @@ type Queryable interface {
 //----------------------------------------
 // MultiStore
 
+// StoreUpgrades defines a series of transformations to apply the multistore db upon load
+type StoreUpgrades struct {
+	Renamed []StoreRename `json:"renamed"`
+	Deleted []string      `json:"deleted"`
+}
+
+// StoreRename defines a name change of a sub-store.
+// All data previously under a PrefixStore with OldKey will be copied
+// to a PrefixStore with NewKey, then deleted from OldKey store.
+type StoreRename struct {
+	OldKey string `json:"old"`
+	NewKey string `json:"new"`
```

Comment:
```suggestion
	NewKey string `json:"new_key"`
```

Reply:
See above, I would rather change variable names than json names. But actually prefer to leave as is. If other core devs agree, I will change as you like.

Reply:
https://github.com/cosmos/cosmos-sdk/pull/4724#discussion_r310770896

Reply:
Done